### DISCUSSION TITLE
MFS is not functioning

### DISCUSSION THREAD

**Discussion by yuriks59** [flybywiresim/aircraft#4246] [General]
I installed the latest A32NX update, everything would be fine, but MFS does not function and the time on the clock does not start. Who will tell you how to fix it. I put the old mod, everything works.

**Comment by CptBayer** [comment]
how to get the old mod installed, better the version prior the update?  Have you had a backup from the prior version?

greets Klaus

**Reply by yuriks59** [reply]
Yes, I am already learned and keep the old working version. therefore, I installed it and fly without problems on a working plane.

**Reply by CptBayer** [reply]
you lucky boy :-P ... next time I will do a backup too ....

**Reply by Paytheon** [reply]
Hehe, I've learned to backup the last known good version too. Excellent advice.

**Comment by ExampleWasTaken** [comment]
There is an update for the installer. Download that, then delete the old A32NX folder from your community folder, and reinstall the version you want.

**Reply by CptBayer** [reply]
rock´n roll !!!!

**Reply by yuriks59** [reply]
Yes, I have an installer on my desktop, installed it as you say, still something does not work.

**Reply by ExampleWasTaken** [reply]
@yuriks59 did you follow my instructions?

**Reply by yuriks59** [reply]
I found the reason, it's in the fashion of the pilots in the cockpit, removed the mod and everything worked.

**Comment by jachu1978** [comment]
hello i have the same new update does not work a320 is by default.

**Reply by ExampleWasTaken** [reply]
> @jachu1978  did you follow my instructions?
?

**Reply by jachu1978** [reply]
I have been doing an A320neo update every day for a long time and I do it according to the instructions and it always worked without a problem about this new update yours it stopped working when I install it a320neo is the default not working.
http://foto-hosting.pl/img/b8/56/b6/b856b66335d7cd06afd6ca560adb30093a1efc1d.png
http://foto-hosting.pl/img/42/06/0c/42060ce902a32277bc2c6247463cd4b814802891.png
http://foto-hosting.pl/img/68/f7/1e/68f71ea146d2b1943f9d7390c9f0f089c8547b7e.png

http://foto-hosting.pl/img/09/15/32/091532e2222e4f8535bfd81020be8f260a15d4c4.png
this mod is from before last update and everything works new updates don't work




everything is fine on my side as well

**Reply by ExampleWasTaken** [reply]
As far as I understand your post, you have to delete the `A32NX` folder **AND** the `flybywire-aircraft-a320-neo` folder from your community folder. Then make sure you are on installer version 1.1.4. You can check the version of your installer in the installer settings. After you verified that you have the latest version of the installer (1.1.4) you can then download the aircraft-version of your liking again.

**Comment by jachu1978** [comment]
I checked and I have the latest version of the installer and there is still a problem with the new modification. The older version of the update loads without any problems and everything works. The newer version of the update does not load A320 by default

**Reply by ExampleWasTaken** [reply]
Did you delete all old a32nx related folders from your community folder? Also, the development and experimental versions A32NX are now a separate aircraft in sim. You can find them as `FlyByWireSimulations A320neo (LEAP)`. Be aware that the stable still overwrites the default A320neo.

**Reply by lionomega** [reply]
I have the same problem.
No matter how much I delete and install with the latest version, the developer version never installs only the default asobo

**Comment by jachu1978** [comment]
I tested all the possibilities and your new a320 update still does not work, the problem is with you because I see and read the posts of other users who have the same problem. The older version of the latest update works correctly. It looks like mfs2020 does not see this folder with the new update and it becomes default A320neo.

**Reply by ExampleWasTaken** [reply]
I am not sure you understood my post correctly. The development version and the experimental version are no longer overlayed on top of the default A320. They are now a separate aircraft. Look for an A320 named:
```
FlyByWireSimulations 
A320neo (LEAP)
```
in your aircraft selector.

**Reply by ExampleWasTaken** [reply]
There seems to be an issue where the A32NX shows up like the default A320. The current workaround for that is to check whether it has the FlyByWire livery or not. If it does, it's the A32NX.

**Comment by jachu1978** [comment]
should I copy a320neo here ?
https://imgur.com/obpDKQf

**Reply by ExampleWasTaken** [reply]
Here's a detailed list of what actions are required to get the aircraft running.

1. Empty your community folder completely
2. [Download](https://api.flybywiresim.com/installer) the official FlyByWire installer.
3. Close Microsoft Flight Simulator 2020
4. Open the installer and select the version you want to download by clicking on it.
5. Next click `install`.
6. Once the installation is complete you are ready to fly.

### Liveries
Liveries made for the default A320neo that ships with Microsoft Flight Simulator 2020 are no longer going to work.
Liveries must explicitly state that they support the A32NX.
An extensive list of liveries that support the A32NX can be found [here](https://flightsim.to/c/liveries/flybywire-a32nx/).
These liveries must be put into the Community folder just like any other livery too.

**Comment by jachu1978** [comment]
Hello, I understand what you wrote only if I install the updates with the installer, it installs to the community folder from the vending machine, I would ask for more detailed instructions for this installation or how to prevent the update from installing in the community folder.

<a href="https://www.fotosik.pl/zdjecie/261944b535f8345d" target="_blank"><img src="https://images89.fotosik.pl/490/261944b535f8345dmed.png" border="0" alt="" /></a>

**Reply by ExampleWasTaken** [reply]
Does the path you marked point to your community folder? If it does, everything is fine. The aircraft will then be installed correctly. If the path you marked is does not point to your community folder then change it to your community folder.

**Comment by jachu1978** [comment]
but I do install a320 with your installer the file that is downloaded automatically saves in the community folder because I can see it and it still does not work as it should or there is a second community foloder or I don't understand anything anymore

**Comment by jachu1978** [comment]
The situation is under control, I do not fully understand you, but it's ok, you just had to go in and choose A320neo

<a href="https://www.fotosik.pl/zdjecie/84c3f819419fa35d" target="_blank"><img src="https://images89.fotosik.pl/490/84c3f819419fa35dmed.png" border="0" alt="" /></a
problem solved :)  thank you very much buddy ( ExampleWasTaken ) !! :)

**Comment by yuriks59** [comment]
finally figured out everything worked as it should, although not all liveries work

**Reply by ExampleWasTaken** [reply]
@yuriks59 read what I wrote here:
https://github.com/flybywiresim/a32nx/discussions/4246#discussioncomment-572441

**Reply by yuriks59** [reply]
Everything is fine, I have everything working with the latest version of Fly By Wire Installer.
